homeassistant/components/mqtt/mixins.py

```diff
@@ -10,6 +10,7 @@
 
 import voluptuous as vol
 
+from homeassistant.components.repairs import IssueSeverity, async_create_issue
```

Comment:
We can import these from `helpers.issue_registry` now.

Reply:
Done